keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -644,6 +655,12 @@ spec:
     port: 80
   sessionAffinity: None
 status:
+  status:
```

Comment:
should we provide/define some common reasons when the ready condition is true/false/unknown? similar to [gateway api](https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1/gateway_types.go#L897-L971)